portal/hooks/useEstimateFees.ts

```diff
@@ -6,6 +6,10 @@ import { useIsConnectedToExpectedNetwork } from './useIsConnectedToExpectedNetwo
 const defaultBlockCount = 4
 const defaultOverEstimation = 1
 
+/**
+ * Calculates the average of an array of bigints.
+ * Returns the average as a stringified number.
+ */
 const mean = function (rewards: bigint[] = []) {
   if (rewards.length === 0) {
     return '0'
```

Comment:
Given this function is only used locally, this line will never run, as in line 71 where this function is called, we're ensuring `rewards` are always defined and with `.length > 0`. The same applies to the default ` = []`